examples/WordClassifier.hs

```diff
@@ -27,9 +27,9 @@ import qualified Streamly.Prelude as S
 import           System.Environment (getArgs)
 
 instance (Enum a, Storable a) => Hashable (A.Array a) where
-    hash arr = runIdentity $ IUF.fold A.read IFL.rollingHash arr
-    hashWithSalt salt arr = runIdentity $
-        IUF.fold A.read (IFL.rollingHashWithSalt salt) arr
+    hash arr = fromIntegral $ runIdentity $ IUF.fold A.read IFL.rollingHash arr
```

Comment:
The Int64 now created this odd incompatibility with Hashable class. Not sure if that's a problem.